impl/wsWslay/NWebsocketWslay.cpp

```diff
@@ -306,6 +304,10 @@ namespace Nakama {
             return;
         }
 
+        if (_state == State::RemoteDisconnect) {
```

Comment:
I'd add comment here that `_state` can change since previous check in the same tick() call